graphql_compiler/scripts/generate_test_sql/animals.py

```diff
@@ -0,0 +1,75 @@
+import random
+
+from .species import SPECIES_LIST
+from .utils import (create_edge_statement, create_name, create_vertex_statement, get_uuid,
+                    strip_index_from_name)
+
+
+random.seed(0)
```

Comment:
Let's not mutate shared global state on import. Do this in the entry point function for the module instead.